tests/main/services-install-hook-can-run-svcs/task.yaml

```diff
@@ -1,6 +1,15 @@
 summary: Check that install hooks in snaps can start services
 
+environment:
+  FLAGS/enable: --enable
+  FLAGS/none: ""
```

Comment:
now that I looked at test logs  none is not very self-explanatory here,  s/none/one-off/ ? or something, anyway I'm going to land this but maybe on master we should follow-up with a better naming

Reply:
What about "no-flag" ?